doc/install.rst

```diff
@@ -124,6 +121,35 @@ Some dependencies can be specified as follows. Note only some of these support
   Path to Python used by :code:`sphinx-build`. Default: shebang of
   :code:`sphinx-build`.
 
+Some dependencies can be specified as follows. Note only some of these support
+:code:`--with-FOO=no`, as listed.
+
+Optional bind selection: :code:`--bind-FOO`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, :code:`ch-run` binds the minimal number mount points needed to run.
+You can add more files with:
+
+:code:`--bind-path={PATH[:PREFIX}`
+  Add file from path to to ch-run optional default run-time binds.
+
+    * Path to host file. Optionally bind at prefix if set; otherwise,
+      :code:`/usr/local/lib`
```

Comment:
I'm pretty sure this (i.e., arbitrary default bind-mounts) can be implemented, but I'm not sure offhand how to do it. If there's not an immediate use case, maybe leave this for future work?